apps/web/src/common/components/editor/extensions/image/plugins/drop-image.ts

```diff
@@ -94,8 +94,8 @@ export const dropImagePlugin = (upload: ImageUploader<any>, imgFileDataMap: Map<
                             src: downloadUrl,
                             'file-id': fileId,
                         });
-                        // const transaction = view.state.tr.insert(coordinates.pos, node);
-                        const transaction = view.state.tr.setNodeMarkup(coordinates.pos, schema.nodes.image, node.attrs);
+                        const loadingPos = view.state.selection.anchor - 1; // get the position of the loading node
+                        const transaction = view.state.tr.setNodeMarkup(loadingPos, schema.nodes.image, node.attrs);
```

Comment:
description: At TextEditor component, this is drag&drop plugin for image. 
While uploading the image, loading node will be appeared, and when upload finished, the image should be appeared.
To do that, replace loading node with image node by `loadingPos`.

Reply:
😮